python/src/opendp/_extrinsics/_make_np_eigenvalues/__init__.py

```diff
@@ -10,7 +10,12 @@ def make_np_eigenvalues(input_domain: Domain, input_metric: Metric) -> Transform
     :param input_domain: instance of `_np_sscp_domain(size=_, num_columns=_)`
     :param input_metric: instance of `symmetric_distance()`
     """
-    import numpy as np  # type: ignore[import]
+    try:
+        import numpy as np # type: ignore[import-not-found]
+    except ImportError:
+        raise ImportError(
+            "The optional install numpy is required for this functionality"
+        )
```

Comment:
Can you abstract this with a helper function? For reference, see how pandas does it:
https://github.com/pandas-dev/pandas/blob/main/pandas/compat/_optional.py

Reply:
Will do: Don't think we need all the options in that example, but good to know this is a pattern.

Reply:
Agreed, Pandas' approach is a bit more beefy than we need.